src/3d/qgspointcloudlayerchunkloader_p.h

```diff
@@ -101,9 +101,9 @@ class QgsPointCloudLayerChunkLoader : public QgsChunkLoader
 
     /**
      * Constructs the loader
-     * QgsPointCloudLayerChunkLoader takes ownership over symbol
+     * \note since QGIS 3.28 the context doesn't take ownership over the passed symbol \a symbol
```

Comment:
If we aren't cloning the symbol then we're going to run into all sorts of threading issues -- the symbols aren't thread safe and you'll get crashes if you modify the symbol in the main thread while the loader is doing stuff with them in a background thread.....